x/upgrade/module.go

```diff
@@ -64,11 +64,6 @@ func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *g
 	}
 }
 
-// GetQueryCmd returns the CLI query commands for this module
-func (AppModuleBasic) GetQueryCmd() *cobra.Command {
```

Comment:
is it okay to remove this method completely? I observed this method was returning `nil` in other modules.

Reply:
Yes, it was the case, but the interface has been simplified for removing that unnecessary boilerplate (https://github.com/cosmos/cosmos-sdk/pull/16890)